query/stat_processor.go

```diff
@@ -9,11 +9,21 @@ import (
 
```

Comment:
not sure any of the changes in this file are needed. it would also help reduce the scope of what I need to review if we remove the subjective refactorings and leave it mostly Clickhouse changes

Reply:
Ok, agreed.
Summary: updated